### ISSUE TITLE
manually set "Setup parameters" does not accept  linux paths

### ISSUE THREAD

**Issue by HoWilgh** [ndrs92/DesignSPHysics#11]
Hi,

I installed your workbench/macro via the addon-manager on Ubuntu 16.04. I run the DesignSPHysics.FCMarco and tried to set the "Setup parameters" (Setup plugin).
The problem ist that your dialog does not accept the paths like 
"/home/USER/.FreeCAD/Mod/DesignSPHysics/dualsphysics/EXECS" where e.g. "GenCase4_linux64" is actually stored. It gives the following warning while ignoring that the correct/appropriate file is in the given folder:
'One or more of the executables in the setup is not correct. Check plugin setup to fix missing binaries.'

Maybe you are looking for .exe files while I use linux.
BR,
HoWil

**Comment by ndrs92** [comment]
Hi, thank you for the issue. 

I'll test it today in GNU/Linux and polish the setup window a little.

**Comment by ndrs92** [comment]
Hi again,

I installed in a clean Ubuntu VM DesignSPHyisics via the addon-manager too.

It accepts paths the one you suggested (I used that, in fact), the thing is that by default, after downloading DesignSPHysics with the addon manager, the permissions of the executables that were downloaded **does not** include execution permissions.

**So the easy fix is to chmod the entire executable directory adding execution permissions.** 

However, I'll add a more descriptive error dialog on the setup, giving information in things like this.  I'll close this issue once its done.
